### ISSUE TITLE
Verification with emoji: Emoji glyphs are not the same than on web and android

### ISSUE THREAD

**Issue by BillCarsonFr** [element-hq/element-x-ios#2242]
### Steps to reproduce

1. Verify with emoji

#### What did you expect?

The emoji should match but they don't

#### What happened instead?
![image](https://github.com/element-hq/element-x-ios/assets/9841565/8c0533a4-8dd3-41db-928d-a35761429b78)

<img width="490" alt="image" src="https://github.com/element-hq/element-x-ios/assets/9841565/a60ea0c9-9e75-410e-826a-8e293e8b8dfe">

They don't match
<img width="334" alt="image" src="https://github.com/element-hq/element-x-ios/assets/9841565/fbe6d063-f8cf-4395-8471-a31ebc675728">

see https://github.com/element-hq/element-x-android/pull/1965

### Will you send logs?

Yes

**Comment by brianredbeard** [comment]
If my understanding is correct, In order to properly resolve this would require vendoring a set of libre/freely licensed emojis.

**Comment by stefanceriu** [comment]
> In order to properly resolve this would require vendoring a set of libre/freely licensed emojis

Yes, moment in which I would just implement [the decimal variant](https://spec.matrix.org/unstable/client-server-api/#sas-method-decimal) and save everybody a lot of pain.
